src/SnackbarProvider.js

```diff
@@ -189,7 +191,8 @@ class SnackbarProvider extends Component {
     };
 
     render() {
-        const { children, maxSnack, ...props } = this.props;
+        const { children, maxSnack, classes, anchorOrigin = {}, className, ...props } = this.props,
```

Comment:
You didn't update the `proptypes`. (but i'm not sure you have to update `index.d.ts`)